File: java/chaincue-real-estate/src/main/java/com/chaincuerealestate/chaincuerealestate/repositories/BrokerRepository.java
package com.chaincuerealestate.chaincuerealestate.repositories;

import com.chaincuerealestate.chaincuerealestate.domains.Broker;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface BrokerRepository extends JpaRepository<Broker, String> {
}

File: kotlin/chaincue-real-estate-reactive/src/main/kotlin/com/chaincuerealestate/chaincuerealestatereactive/repositories/BrokerRepository.kt
package com.chaincuerealestate.chaincuerealestatereactive.repositories

import com.chaincuerealestate.chaincuerealestatereactive.domains.Broker
import org.springframework.data.repository.kotlin.CoroutineCrudRepository
import org.springframework.stereotype.Repository

@Repository
interface BrokerRepository : CoroutineCrudRepository<Broker, String>

